benchmarks/one_comp_mm_elim_abs/one_comp_mm_elim_abs.info

```diff
@@ -0,0 +1,6 @@
+# Reference calculations made with
+
+./one_comp_mm_elim_abs sample num_samples=1000 adapt delta=0.9 data file=one_comp_mm_elim_abs.data.R id=1 output file=output.1.csv
+./one_comp_mm_elim_abs sample num_samples=1000 adapt delta=0.9 data file=one_comp_mm_elim_abs.data.R id=2 output file=output.2.csv
```

Comment:
How about fixing the random seed?

Reply:
Seeds are misleading for this stuff. It should work regardless of the seeds.

Reply:
but different seeds can really have a big impact on speed things and the sampler results unless you sample a crazy amount of iterations. If you don't want seeds, then at least lets fix `init=0` (if you think that's not good, then a small number is also fine).

... also, I think we fix seeds for all the other examples, so we should be consistent would be my position here. I would prefer the seeds to be set here. Maybe we run this by discourse quickly?

The thing is that seeds will get the variation from run to run down - and that's a good thing for this type of test which we want to run as reproducible as possible.

Reply:
I put seeds back in there for these two models.

I suspect the performance tests are using different run scripts than were here (sir here was using regular non-NUTS hmc). I think we can probably get seeds in whatever scripts are being used there.